fvm/evm/stdlib/contract.cdc

```diff
@@ -9,33 +11,45 @@ contract EVM {
         access(all)
         let bytes: [UInt8; 20]
 
-        /// Constructs a new EVM address from the given byte representation.
+        /// Constructs a new EVM address from the given byte representation
         init(bytes: [UInt8; 20]) {
             self.bytes = bytes
         }
+
+        /// Deposits the given vault into the EVM account with the given address
+        access(all)
+        fun deposit(from: @FlowToken.Vault) {
+            let amount = from.balance
```

Comment:
I would add `from.balance = 0.0` just incase

Reply:
This has been addressed in a follow up PR, #4986, see https://github.com/onflow/flow-go/pull/4986/files#diff-02837edc597afad5bd8f7d1962c129bded25adcc21a9889382c7aaad9a0e5b34L23-R26